--- blitz-1.0.2/doc/doxygen/Makefile.am.orig	2024-04-22 20:05:26.115494996 +0200
+++ blitz-1.0.2/doc/doxygen/Makefile.am	2024-04-22 20:18:29.974581800 +0200
@@ -91,10 +91,6 @@ make-install-dirs:
 	fi
 
 install-data-hook: make-install-dirs
-	@if test -d html; then \
-		echo Installing HTML documentation into $(DESTDIR)$(docdir) ; \
-		$(INSTALL_DATA) html/* $(DESTDIR)$(docdir) ; \
-	fi
 	@if test -f $(PACKAGE).pdf; then \
 		echo Installing PDF manual into $(DESTDIR)$(docdir) ; \
 		$(INSTALL_DATA) $(PACKAGE).pdf $(DESTDIR)$(docdir) ; \
