diff -Nurb blitz-0.8.orig/m4/ac_compiler_specific_header.m4 blitz-0.8.new/m4/ac_compiler_specific_header.m4
--- blitz-0.8.orig/m4/ac_compiler_specific_header.m4	2004-10-13 18:14:04.000000000 +0200
+++ blitz-0.8.new/m4/ac_compiler_specific_header.m4	2005-05-21 10:24:42.211975904 +0200
@@ -22,7 +22,7 @@
             COMPILER_SPECIFIC_HEADER="hp/bzconfig.h" ;;
   CC)       AX_PREFIX_CONFIG_H([blitz/sgi/bzconfig.h],[BZ])
             COMPILER_SPECIFIC_HEADER="sgi/bzconfig.h" ;;
-  g++)      AX_PREFIX_CONFIG_H([blitz/gnu/bzconfig.h],[BZ])
+  g++|*-*-*-g++)      AX_PREFIX_CONFIG_H([blitz/gnu/bzconfig.h],[BZ])
             COMPILER_SPECIFIC_HEADER="gnu/bzconfig.h" ;;
   pgCC)     AX_PREFIX_CONFIG_H([blitz/pgi/bzconfig.h],[BZ])
             COMPILER_SPECIFIC_HEADER="pgi/bzconfig.h" ;;
